src/qt/transactiondesc.cpp

```diff
@@ -108,9 +110,13 @@ QString TransactionDesc::toHTML(CWallet *wallet, CWalletTx &wtx)
     //
     // From
     //
-    if (wtx.IsCoinBase() || wtx.IsCoinStake())
+    if (wtx.IsCoinBase())
     {
-        strHTML += "<b>" + tr("Source") + ":</b> " + tr("Generated") + "<br>";
+        strHTML += "<b>" + tr("Source") + ":</b> " + tr("Generated in CoinBase") + "<br>";
+    }
+    else if (wtx.IsCoinBase() || wtx.IsCoinStake())
```

Comment:
Should `wtx.IsCoinBase()` be here as well?

Reply:
True, i will remove it.